__tests__/app.test.js

```diff
@@ -96,6 +96,52 @@ describe("/api/articles/:article_id", () => {
             expect(body.msg).toBe("Not Found")
         })
     })
+
+    test("PATCH 200: Responds with 200 OK when a valid patch request is sent", () => {
+
+        const body = {
+            inc_votes: 1
+        }
+
+        return request(app)
+        .patch("/api/articles/1")
+        .send(body)
+        .expect(200)
+        .then(({body})=> {
+            expect(body.updatedRow.votes).toBe(101)
```

Comment:
It might be more useful to test for the entire response here, otherwise we can't be sure that we actually get all of the properties back and don't just get this one row back on the response.